### DISCUSSION TITLE
Explain the existing plugins

### DISCUSSION THREAD

**Discussion by spapas** [django-wiki/django-wiki#1345] [Q&A]
There are a bunch of plugins like GlobalHistory, Help etc. See here:

https://github.com/django-wiki/django-wiki/tree/main/src/wiki/plugins

Can you please explain what each one of them is doing? 

TY

**Comment by benjaoming** [comment]
Thanks for reporting this! I've moved it to an issue, I think it should be in the documentation: https://github.com/django-wiki/django-wiki/issues/1347
